docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -654,6 +654,15 @@ file by adding a comment such as
 # rubocop:enable Layout/LineLength, Style/StringLiterals
 ----
 
+You can also disable entire departments by giving a department name in the comment.
```

Comment:
I'd add another example showing that you can also combine in the comments department references with individual cops.

Reply:
Combined with individual cop